src/DB_Replace.php

```diff
@@ -2,9 +2,140 @@
 
 namespace JustCoded\WP\Composer;
 
-// TODO: create db replace command
-
+use Composer\Script\Event;
+use JustCoded\WP\Composer\Helpers\Array_Helper;
+use JustCoded\WP\Composer\Helpers\Process_Controller;
+use JustCoded\WP\Composer\Helpers\Scripts_Helper;
+use PHP_CodeSniffer\Tokenizers\PHP;
 
+/**
+ * Replace srting in DB
+ *
+ * Usage:
+ *      wp:dbUpdate -- --search='old string' --replace='new string' --method --tables
+ *
+ * Options:
+ *      --search    Old string
+ *      --replace   New string
+ *      --method    Method replace only content columns/string data columns or replace all data columns full/simple (default simple)
+ *      --tables    List of tables separated by comma (default all)
+ *
+ */
 class DB_Replace {
 
+	public static function db_replace( Event $event ) {
+		$args    = Scripts_Helper::parse_arguments( $event->getArguments() );
+		$options = self::checkOptions( $args );
+
+		if ( ! $options || empty( $options['search'] ) ) {
+			self::printError();
+		}
+
+		$replace_param = self::prepareOptions( $options );
+		self::initReplace( $replace_param );
+	}
+
+	/**
+	 * Print error MSG
+	 *
+	 * @throws \ReflectionException
+	 */
+	protected static function printError() {
+		echo PHP_EOL;
+		$rc = new \ReflectionClass( get_class() );
+		print_r( '========================================================' );
+		echo PHP_EOL;
+		print_r( '|| Something wrong. Read documentation and try again. ||' );
+		echo PHP_EOL;
+		print_r( '========================================================' );
+		echo PHP_EOL;
+		print_r( $rc->getDocComment() );
+		echo PHP_EOL;
+		die;
+	}
+
+	/**
+	 * @param array $replace_param
+	 */
+	protected static function initReplace( $replace_param = array() ) {
+		self::loadWp();
+		global $wpdb;
+
+		$controller = new Process_Controller();
```

Comment:
It's not correct to call helper as "Controller". 

If you want to use wp host update class "as is" - it's better to create composer.json file inside wp-host-update repository, then manually require files from vendor/justcoded/wp-host-update/app/* and create class instances. In this case you don't need to duplicate any code.